stories/cosmoz-textarea.stories.js

```diff
@@ -17,5 +17,20 @@ const basic = () => html`
 			.errorMessage=${'Something is wrong!'}
 			.maxRows=${2}
 		></cosmoz-textarea>
-	`;
-export { basic, error };
+	`,
+	contour = () => html`<style>
+			cosmoz-textarea {
+				--cosmoz-input-color: #aeacac;
+				--cosmoz-input-border-radius: 4px;
+				--cosmoz-input-padding: 12px;
+				--cosmoz-input-line-display: none;
+				--cosmoz-input-contour-size: 1px;
+				--cosmoz-input-label-translate-y: 35%;
+				--cosmoz-input-input-translate-y: 8px;
+			}
+		</style>
+		<cosmoz-textarea
+			.label=${'Choose color'}
+			.value=${'Red\nGreen\nBlue'}
```

Comment:
Could you put something more appropriate for a textarea?

Reply:
Yes, I changed the label and put a good old "Lorem ipsum" as a placeholder